plugin-server/src/utils/db/db.ts

```diff
@@ -658,11 +658,10 @@ export class DB {
         teamId: number,
```

Comment:
Are these changes to what looks to be "group cache" in this file related to this PR?

Reply:
Nope, these were cherry picked from bf4728c63117fa246f261377b010d03c48808432 to fix some tests

Reply:
Just merged with master since some other stuff changed too

Reply:
Just merged with master since some other stuff changed too